backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5709110_sys_gh16561_AD_Printer_Config_UC_Workspace_HostKey.sql

```diff
@@ -0,0 +1,24 @@
+-- 2023-10-29T18:25:14.726Z
+INSERT INTO AD_Index_Table (AD_Client_ID,AD_Index_Table_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,EntityType,ErrorMsg,IsActive,IsUnique,Name,Processing,Updated,UpdatedBy,WhereClause) VALUES (0,540772,0,540637,TO_TIMESTAMP('2023-10-29 19:25:14','YYYY-MM-DD HH24:MI:SS'),100,'D','Pro Arbeitsplatz und Hostkey ist nur ein Datensatz erlaubt','Y','Y','AD_Printer_Config_UC_Workspace_HostKey','N',TO_TIMESTAMP('2023-10-29 19:25:14','YYYY-MM-DD HH24:MI:SS'),100,'IsActive=''Y''')
+;
+
+-- 2023-10-29T18:25:14.728Z
+INSERT INTO AD_Index_Table_Trl (AD_Language,AD_Index_Table_ID, ErrorMsg, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Index_Table_ID, t.ErrorMsg, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Index_Table t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Index_Table_ID=540772 AND NOT EXISTS (SELECT 1 FROM AD_Index_Table_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Index_Table_ID=t.AD_Index_Table_ID)
+;
+
+-- 2023-10-29T18:26:58.801Z
+UPDATE AD_Index_Table_Trl SET ErrorMsg='Only one record is permitted per workspace and host key', IsTranslated='Y',Updated=TO_TIMESTAMP('2023-10-29 19:26:58','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Index_Table_ID=540772 AND AD_Language='en_US'
+;
```

Comment:
'Only one active record is permitted per workspace and host key' (IMHO we don't need host key for workplaces)

Reply:
done